worker/src/Channel/Request.cpp

```diff
@@ -140,13 +157,15 @@ namespace Channel
 		if (data.is_structured())
 			jsonResponse["data"] = data;
 
-		this->channel->Send(jsonResponse);
+		Request::channel->Send(jsonResponse);
 	}
 
 	void Request::Error(const char* reason)
 	{
 		MS_TRACE();
 
+		MS_ASSERT(Request::channel != nullptr, "channel unset");
+
```

Comment:
Let's replace it with `MS_ASSERT(Request::channel, "channel unset");` and let's remove the empty line between both `MS_ASSERT()`s.